templates/test/dev/cluster-template-custom-builds.yaml

```diff
@@ -375,6 +384,39 @@ spec:
           New-Item -ItemType Directory -Force -Path C:\tmp\
         path: C:/create-temp-folder.ps1
         permissions: "0744"
+      - content: |
```

Comment:
Can we set `$ErrorActionPreference Stop` at the beginning of this script?
I'm worried that if we don't and there is an issue with any of the packages they might get skipped over and we won't test what we think we are testing.

Reply:
We should do this everywhere TBH